### PR TITLE
Added new boiler model implementations

### PR DESCRIPTION
**Pull request by arnoutaertgeerts**
New boiler model implementations as described in https://github.com/open-ideas/IDEAS/issues/45#.

### CODE REVIEW

**Comment by saroele** [comment]
My commits are added now, if unittests are fine, I think we can merge.

**Comment by Mathadon** [comment]
I'll check unit tests

File: IDEAS/Utilities/Psychrometrics/Functions/BaseClasses/Examples/DewPointTemperatureDerivativeCheck.mo [hunk 1]
```diff
@@ -12,7 +12,7 @@ equation
     x=IDEAS.Utilities.Psychrometrics.Functions.pW_TDewPoi(  T=time*uniCon);
     der(y)=der(x);
     assert(abs(x-y) < 1E-2, "Model has an error");
-   annotation(                       __Dymola_Commands(file="modelica://IDEAS/Resources/Scripts/Dymola/Utilities/Psychrometrics/Functions/BaseClasses/Examples/DewPointTemperatureDerivativeCheck.mos"
```
Comment (Mathadon) [inline IDEAS/Utilities/Psychrometrics/Functions/BaseClasses/Examples/DewPointTemperatureDerivativeCheck.mo]: Please do not include changes such as these in commits. They are caused by allowing Dymola to save an entire (sub)package. They can be reverted by doing a git checkout on the altered files.

**Comment by Mathadon** [comment]
The new unit tests are badly formatted since they do not produce any results. 
The BesTest unit tests give different results.
Some other unit tests containing boilers and heat pumps also give different results.

Also I get a warning:
```
*** Error: HeatPump_Events.mos does not contain any plot command.
You need to add a plot command to include its
results in the regression tests.
```
but this may be pre-existing.

I propose to look at this next meeting.

**Comment by saroele** [comment]
ok, for thursday then

**Comment by damienpicard** [comment]
Add the <i>y={}</i> in the plot command of the script

**Pull request closed without merging**